### PR TITLE
MSC3980: Dotted Field Consistency

### PR DESCRIPTION
**Pull request by clokep**
[Rendered](https://github.com/matrix-org/matrix-spec-proposals/blob/clokep/dotted-consistency/proposals/3980-dotted-fields-consistency.md)

[FCP tickyboxes](https://github.com/matrix-org/matrix-spec-proposals/pull/3980#issuecomment-1470882783)

### CODE REVIEW

File: proposals/3980-dotted-fields-consistency.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+are currently escaped for consistency.
+
+Through the MSC process the escaping was expanded from just escaping `.` characters
+with a backslash (`\.`) to also escaping backslashes themselves (`\\`). Unfortunately
+that MSC did not propose applying this change *back* to filters for consistency. [^1]
+
+## Proposal
+
+Apply consistent escaping as described in [MSC3873](https://github.com/matrix-org/matrix-spec-proposals/pull/3873)
+to the `event_fields` property of filters (in addition to push rule conditions).
```
Comment (turt2live) [inline proposals/3980-dotted-fields-consistency.md:15]:
```suggestion
to the `event_fields` property of filters.
```

the bit in the brackets confused me, and I think the sentence is fine without. The supporting docs throughout this MSC fix any remaining confusion.

**Comment by turt2live** [comment]
@mscbot fcp merge

**Review by anoadragon453 — APPROVED** [review]
After having a quick browse through the spec, I see no other inconsistencies with backslash handling :+1:

**Comment by turt2live** [comment]
Merged 🎉

**Comment by turt2live** [comment]
Better late than never: we didn't require an implemention on this because the MSC is effectively self-referencing. That's to say it's so trivial that a formal PR/deployment with the MSC's behaviour wouldn't prove anything we don't already know, so the MSC simply existing acts as "implementation".

We talked about it in a public room somewhere (#matrix-spec I think?) stating this.

**Pull request merged**